modules/libcom/src/osi/epicsThread.h

```diff
@@ -437,7 +437,7 @@ class LIBCOM_API epicsThread {
     epicsThread ( const epicsThread & );
     epicsThread & operator = ( const epicsThread & );
     friend void epicsThreadCallEntryPoint ( void * );
-    void printLastChanceExceptionMessage (
+    EPICS_NORETURN void printLastChanceExceptionMessage (
```

Comment:
I have a slight preference for adding decorators macros to the previous line.  Partly for (imo.) readability, and partly for `git-blame`.

```c
EPICS_NORETURN
void printLastChanceExceptionMessage (
```

Reply:
I second that preference. Should it be mentioned in a comment in modules/libcom/src/osi/compilerDependencies.h? That file looks like it has Doxygen comments even though in reality only the file header is one so the file does get processed. We could start them with this macro…

Reply:
What with 
LIBCOM_API EPICS_NORETURN void epicsExit ?
would you prefer

LIBCOM_API
EPICS_NORETURN
void epicsExit

or

LIBCOM_API EPICS_NORETURN
void epicsExit

or

LIBCOM_API
EPICS_NORETURN
void
epicsExit 

or

EPICS_NORETURN
LIBCOM_API void epicsExit ?

Reply:
The second one looks good to me:
```C
LIBCOM_API EPICS_NORETURN
void epicsExit(int status);
```

Reply:
Done.